packages/react-docs/src/pages/components/alert.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ComponentDocs from '../../components/componentDocs';
+import PropTypes from 'prop-types';
+import { Alert, alertVariant, Button } from '@patternfly/react-core';
+import Example from '../../components/example';
+
+const propTypes = {
+  data: PropTypes.any.isRequired
+};
+
+function capitalizeFirstLetter(displayString) {
+  return displayString.charAt(0).toUpperCase() + displayString.slice(1);
+}
+
+const AlertDocs = ({ data }) => (
+  <ComponentDocs data={data}>
+    {Object.keys(alertVariant).map((type, key) => (
+      <Example title={`Notification - ${type}`} key={key + type}>
```

Comment:
Find to leave, but as a heads up.  The adding the type here does not make the key any more resilient. Using the index here without the type has the same effect.  The only reason to avoid using indexes for keys is if your array that is being rendered can be reordered. Keys also only need to be unique within the array that is being rendered. Keys can be shared within the app or the same component so you don't have risks of clashes.

Reply:
Yeah there was an error with missing key and I wanted to make it as unique as possible, I guess `key` is enough.